app/upgrades/v14/upgrades.go

```diff
@@ -21,6 +21,11 @@ func CreateUpgradeHandler(
 			return vm, err
 		}
 
+		// Set the minimum height of a valid consumer equivocation evidence
```

Comment:
Shouldn't this be a part of migration and not in the upgrade handler?

Having it as a migration would also minimize the risk of "somebody just deleted it on accident".

Reply:
It's Gaia specific and not ICS specific. So it should go in the Gaia upgrade handler. It's like setting the value of a SDK module param.

Reply:
I'm just nitpicking and leaving the comment for future reference.

There's also this thing: https://github.com/cosmos/gaia/blob/main/app/app.go#L367. By the looks of it, you could also have a separate handler that executes after `CreateUpgradeHandler`.

The code here would apply the provided values on every upgrade - not a big deal 👍 .